src/page-templates/main-content-page/content-page-template.tsx

```diff
@@ -224,6 +227,44 @@ const ContentPageTemplate: NextPage<ContentPageProps> = ({
         </div>
     );
 
+    const renderExternalExamples = (styles: ThemeUICSSObject = {}) => (
+        <div
+            sx={{
+                display: 'flex',
+                columnGap: [0, 'inc70'],
+                flexDirection: ['column', 'row'],
+                alignItems: ['start', 'center'],
+                ...styles,
+            }}
+        >
+            {githubUrl && (
+                <Button
+                    hasIcon
+                    href={githubUrl}
+                    target="_blank"
+                    variant="secondary"
+                    iconName={ESystemIconNames.EXTERNAL}
+                    sx={{
+                        width: 'auto',
+                    }}
+                >
+                    View Code
+                </Button>
+            )}
+            {liveSiteUrl && (
+                <Link
```

Comment:
@mcolella14 previous implementation was originally using the `<CTALink />` element here, which looks exactly like the Flora `<Link linkIcon="arrow"` so I used that here. Was wondering the reasoning for creating the custom `<CTALink />` component? Looks like it's used only in one or two other places in the codebase.

Reply:
Yeah I think we originally had some more styling or functionality on it, but then removed it and didn't refactor, so I'm fine with just taking that out.